### PR TITLE
feat: create profile screen

### PR DESCRIPTION
**Pull request by HuziMuzi**
image

### CODE REVIEW

File: src/components/MemoryCardsList/MemoryCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {TouchableOpacity, View} from 'react-native';
+import {CustomText} from '@src/components/ui/ReadyStyles/Boxes/index';
+import {Color} from '@theme/colors';
+import {styles} from '@src/components/MemoryCardsList/MemoryCard/styles';
+
+type item = {
```
Comment (antoniosk10) [inline src/components/MemoryCardsList/MemoryCard/index.tsx:7]: Item

File: src/utils/randomColor.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  Color.LightGreen,
+  Color.Violet,
+  Color.LightSeagreen,
+  Color.LightSalmon,
+  Color.Lightpink,
+  Color.Blue,
+  Color.BlueLight,
+];
+
+export const randomColors = () => {
```
Comment (antoniosk10) [inline src/utils/randomColor.ts:13]: getRandomColors

File: src/components/MemoryCardsList/styles.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import {StyleSheet} from 'react-native';
+
+type StylesPropsType = 'row' | 'column';
+
+export const styles = (props?: StylesPropsType) =>
```
Comment (antoniosk10) [inline src/components/MemoryCardsList/styles.ts:5]: getStyles

File: src/components/MemoryCardsList/index.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+import {MemoryCard} from '@src/components/MemoryCardsList/MemoryCard/index';
+import LottieView from 'lottie-react-native';
+import Animated, {FadeInDown} from 'react-native-reanimated';
+
+export const MemoryCardsList = () => {
+  const fakeMemoryCard = [...Array(6)].map((_, i) => ({
+    id: i,
+    title: `Pack ${i + 1}`,
+    description: 'Lorem ipsum dolor.',
+  }));
```
Comment (antoniosk10) [inline src/components/MemoryCardsList/index.tsx:14]: move it out from component. Either on line 8 or into separate file

Comment (antoniosk10) [inline src/components/MemoryCardsList/index.tsx:14]: move it out from component. Either on line 8 or into separate file

File: src/components/MyQuizzes/ListQuizzes/index.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import {imagesDifficulty} from '@src/utils/imagesDifficulty';
+
+export const ListQuizzes = () => {
+  const difficultyArray = ['light', 'medium', 'hard'];
+  const fakeQuizzes = [...Array(10)].map((el, i) => ({
+    id: i,
+    title: 'Node',
+    difficulty: difficultyArray[Math.floor(Math.random() * difficultyArray.length)],
+    countQuestion: [...Array(Math.floor(Math.random() * (30 - 10) + 10))],
+  }));
```
Comment (antoniosk10) [inline src/components/MyQuizzes/ListQuizzes/index.tsx:13]: move it out from component

File: src/components/MyQuizzes/ListQuizzes/index.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {FlatList, ImageBackground, Text, TouchableOpacity, View} from 'react-native';
+import {styles} from '@src/components/MyQuizzes/ListQuizzes/styles';
+import {imagesDifficulty} from '@src/utils/imagesDifficulty';
+
+export const ListQuizzes = () => {
+  const difficultyArray = ['light', 'medium', 'hard'];
```
Comment (antoniosk10) [inline src/components/MyQuizzes/ListQuizzes/index.tsx:7]: if it is constant value, it should be in uppercase, DIFFICULTY_ARRAY

**Review by KhaninPasha0488 — APPROVED** [review]

**Review by antoniosk10 — APPROVED** [review]

**Pull request merged**